students/laura-hernandez/ejercicio-1/src/index.ts

```diff
@@ -0,0 +1,47 @@
+const recursiveFibonacci = (n: number) => {
```

Comment:
missing docs. Public API should have documentation using JSDoc, describing what the function does, params and return. The code inside should be self commented